src/analysis/windowed_indicators.jl

```diff
@@ -314,8 +325,31 @@ struct SlidingWindowResults{TT, T<:Real, X<:Real, XX<:AbstractVector{X},
     config::W
 end
 
+"""
+    SegmentWindowResults
+
+A struct containing the output of [`estimate_indicator_changes`](@ref) used with
+[`SegmentWindowConfig`](@ref). It can be used for further analysis, visualization,
+or given to [`significant_transitions`](@ref).
+
+It has the following fields that the user may access
+
+- `x`: the input timeseries.
+- `t`: the time vector of the input timeseries.
+
+- `x_indicator::Vector{Matrix}`, with `x_indicator[k]` the indicator timeseries (matrix
+   with each column one indicator) of the `k`-th segment.
+- `t_indicator::Vector{Vector}`, with `t_indicator[k]` the time vector of the indicator
+  timeseries for the `k`-th segment.
+
+- `x_change::Matrix`, the change metric values with `x[k, i]` the change metric of the
+  `i`-th indicator for the `k`-th segment.
+- `t_change`, the time vector of the change metric.
+
+- [`config::SegmentWindowConfig`](@ref), used for the analysis.
+"""
```

Comment:
Should clarify what I tried to convey when describing how the results of a segment analysis are structured!